apps/checkout-app/frontend/components/templates/CustomizationDetails/CustomizationDetails.tsx

```diff
@@ -43,7 +46,31 @@ const CustomizationDetails: React.FC<CustomizationDetailsProps> = ({
   onSubmit,
 }) => {
   const classes = useStyles();
-  const { control, handleSubmit: handleSubmitForm, formState } = useForm();
+  const {
```

Comment:
This whole part seems like strictly settings related thing, maybe it'd be worth to have it as a separate hook? Also, is there any reason this is not in `CheckoutPreviewFrame`?

Reply:
I don't think I understand. My question is why is this logic not in the CheckoutPreviewFrame component, since it's clearly passed there anyway, not used anywhere else and the logic is strictly related to it.

Reply:
I wanted to make `CheckoutPreviewFrame` component only responsible for displaying iframe with checkout, which could accept ready to send settings to that iframe in props. I've changed it to the hook returning preview seetting. Is it clear now?